templates/ehub.json

```diff
@@ -140,59 +132,9 @@
                         {
                             "name": "WEBSITE_NODE_DEFAULT_VERSION",
                             "value": "10.14.1"
-                        },
-                        {
-                            "name": "APP_SUBSCRIPTION_ID",
-                            "value": "[variables('subscriptionId')]"
-                        },
-                        {
-                            "name": "APP_RESOURCE_GROUP",
-                            "value": "[variables('resourceGroupName')]"
-                        },
-                        {
-                            "name": "APP_TENANT_ID",
-                            "value": "[variables('tenantId')]"
-                        },
-                        {
-                            "name": "APP_LOG_EHUB_CONNECTION",
-                            "value": "[if(equals(parameters('Event Hub Connection String'), ''), listKeys(resourceId('Microsoft.EventHub/namespaces/AuthorizationRules', variables('ehubNewNsName'), 'RootManageSharedAccessKey'), '2017-04-01').primaryConnectionString, parameters('Event Hub Connection String'))]"
-                        },
-                        {
-                            "name": "APP_LOG_EHUB_NAME",
-                            "value": "[variables('ehubName')]"
-                        },
-                        {
-                            "name": "APP_LOG_EHUB_CONSUMER_GROUP",
-                            "value": "[variables('ehubConsumerGroup')]"
-                        },
-                        {
-                            "name": "APP_LOG_EHUB_RESOURCE_GROUP",
-                            "value": "[variables('ehubResourceGroup')]"
-                        },
-                        {
-                            "name": "APP_DL_CONTAINER_NAME",
-                            "value": "[variables('dlContainerName')]"
-                        },
-                        {
-                            "name": "DL_BLOB_PAGE_SIZE",
-                            "value": "100"
-                        },
-                        {
-                            "name": "APP_STATS_QUEUE_NAME",
-                            "value": "[variables('statsQueueName')]"
                         }
                     ],
                     "connectionStrings": [
-                        {
-                            "name": "APP_CLIENT_ID",
-                            "type": "Custom",
-                            "connectionString": "[parameters('App Client ID')]"
-                        },
-                        {
-                            "name": "APP_CLIENT_SECRET",
```

Comment:
https://github.com/alertlogic/al-azure-collector-js/blob/master/updater.js#L31

Does the above need the same change as
https://github.com/alertlogic/al-azure-collector-js/blob/master/master.js#L113-L114

Reply:
MSI in updater is handle slightly different from master. please see my previous comment.